frodo.js

```diff
@@ -1,6 +1,6 @@
-/**
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.